src/v/cluster/health_monitor_backend.h

```diff
@@ -232,6 +236,8 @@ class health_monitor_backend {
     ss::sharded<features::feature_table>& _feature_table;
     ss::sharded<partition_leaders_table>& _partition_leaders_table;
     ss::sharded<topic_table>& _topic_table;
+    ss::sharded<node_status_table>& _node_status_table;
+    std::unique_ptr<config_i> _config;
```

Comment:
naming nit: can we use a more descriptive name, config_api or something along those lines.